examples/lab-robust/v2/qwiklabs.yaml

```diff
@@ -1,32 +1,16 @@
-entity_type: Lab
 schema_version: 2
+entity_type: Lab
 
-default_locale: en
-
-title:
-  locales:
-    en: Robust Lab Example
-    es: Ejemplo de Robust Lab
-
-description:
-  locales:
-    en: Seriously the best lab you've ever taken. Bar none.
-    es: En serio, el mejor lab que has tomado. Sin excepción.
+id: robust-bundle-v2
```

Comment:
Bundles shouldn't have an `id` field. We identify bundles with a unique ID but that ID is not part of the bundle itself. Please talk to Josh if you'd like more info on why.

Reply:
Removed.